js/woo.js

```diff
@@ -0,0 +1,1465 @@
+                };
+            }
+            const networkId = this.detectChainSlug (this.safeString (item, 'protocol', ''));
+            derivedNetworksData[code]['networks'][networkId] = {
+                'network_title': this.safeString (item, 'name'),
+                'allow_deposit': this.safeString (item, 'allow_deposit'),
+                'allow_withdraw': this.safeString (item, 'allow_withdraw'),
+                'withdrawal_fee': this.safeString (item, 'withdrawal_fee'),
+                'minimum_withdrawal': this.safeString (item, 'minimum_withdrawal'),
+                'info': { 'networkInfo': item },
+            };
+        }
+        // combine as final step.
+        const keys = Object.keys (derivedCurrenciesData); // keys and items amount in derivedCurrenciesData and derivedNetworksData are same
+        for (let i = 0; i < keys.length; i++) {
+            const code = keys[i]; // keys are alraedy safeCurrencyCode-d
+            const currencyInfo = derivedCurrenciesData[code];
+            const currencyNetworks = derivedNetworksData[code];
+            const currencyMerged = this.deepExtend (currencyInfo, currencyNetworks);
+            // if response1 each item contains only one child, then they are the same network . Quite bad behavior of their API for this matter, but as-is: the only solution is this as I think at this moment.
+            const name = this.safeString (currencyMerged, 'fullname');
+            const isActive = this.hasChildWithKeyValue (currencyNetworks, 'allow_deposit', 1) && this.hasChildWithKeyValue (currencyNetworks, 'allow_withdraw', 1);
+            result[code] = {
+                'id': code,
+                'name': name,
+                'code': code,
+                'precision': undefined, // TODO: different networks have different precision
+                'info': { 'currencyInfo': currencyInfo, 'networkInfo': currencyNetworks },
+                'active': isActive,
+                'fee': undefined, // TODO
+                'networks': currencyMerged['networks'],
+                'limits': {
+                    'deposit': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'withdraw': {
+                        'min': undefined, // TODO
+                        'max': undefined, // TODO
+                    },
+                },
+            };
+        }
+        return result;
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'order_type': type.toUpperCase (),
+            'side': side.toUpperCase (),
+        };
+        if (price !== undefined) {
+            request['order_price'] = price; // TODO:   this.priceToPrecision (symbol, price);  --> rounds-dow 1.01 to 1.0 , for WOO_USDT
```

Comment:
We have to use priceToPrecision with a proper precision value set on the WOO market.